apps/src/templates/manageStudents/ManageStudentsTable.jsx

```diff
@@ -107,7 +107,7 @@ export const sortRows = (data, columnIndexList, orderList) => {
   }
   addRows = orderBy(addRows, columnIndexList, orderList);
   newStudentRows = orderBy(newStudentRows, columnIndexList, orderList);
-  studentRows = orderBy(studentRows, columnIndexList, orderList);
+  studentRows = studentRows.reverse();
```

Comment:
Is it guaranteed that the `studentRows` data is already sorted by name and creation time, but just in the reversed order?

Reply:
I found a case where it isn't the case you're right. That was the bug I needed to fix, the upcoming commit will fix it by specifying the sort order so that we can guarantee the sorting is consistent.